docs/product-installation-guide/src/main/asciidoc/clustering-ps-configure-domain-proc.adoc

```diff
@@ -97,5 +138,18 @@ The following dialects are supported:
 ** PostgreSQL: `org.hibernate.dialect.PostgreSQL82Dialect`
 ** PostgreSQL plus: `org.hibernate.dialect.PostgresPlusDialect`
 ** Sybase: `org.hibernate.dialect.SybaseASE157Dialect`
++
+. Save the `domain.xml` file.
 
 
+. Complete the following steps in the `host.xml` file on the master node and in the `host-slave.xml` file on each slave node:
+.. Open the `_EAP_HOME_/domain/configuration/host.xml` or `_EAP_HOME_/domain/configuration/host-slave.xml` file in a text editor.
+.. Add the following property to the `<system-properties>` element:
++
+[source]
+----
+<system-properties>
+   <property name="org.jbpm.ejb.timer.tx" value="true"/>  
```

Comment:
This property could be added to domain.xml (point 7 in this chapter) as it is same for all servers.